src/components/ChordSheetEditor.jsx

```diff
@@ -1,33 +1,28 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 
+import store from '../store/index';
```

Comment:
You don't need to import the redux store here. The top level `<Provider>` already passes it down. 

You can use the `mapDispatchToProps` argument of `connect` to connect your redux actions to the store.
```
const mapDispatchToProps = {
  setChordSheet, setSelectionRange
};

connect(mapStateToProps, mapDispatchToProps)(ChordSheetEditor);
```

which is shorthand for:
```
function mapDispatchToProps(dispatch) {
  return {
    setChordSheet: (chordSheet) => dispatch(setChordSheet(chordSheet)),
    setSelectionRange: (start, end) => dispatch(setSelectionRange(start, end)),
  }
}

const mapDispatchToProps = {
  setChordSheet, setSelectionRange
};
```